lms/djangoapps/mobile_api/urls.py

```diff
@@ -12,4 +12,8 @@
     url(r'^my_user_info', my_user_info),
     url(r'^video_outlines/', include('mobile_api.video_outlines.urls')),
     url(r'^course_info/', include('mobile_api.course_info.urls')),
+    url(r'^groups/', include('mobile_api.groups.urls')),
+    url(r'^courses/', include('mobile_api.courses.urls')),
+    url(r'^friends/', include('mobile_api.friends.urls')),
```

Comment:
I would like to see the above 3 endpoints 'groups', 'courses', and 'friends' embedded within a 'social' endpoint - as they are specific to social and not first class endpoints for mobile_api.